app/src/main/java/net/gsantner/opoc/frontend/filebrowser/GsFileBrowserDialog.java

```diff
@@ -149,9 +149,10 @@ public void onViewCreated(final View root, final @Nullable Bundle savedInstanceS
 
         root.setBackgroundColor(rcolor(_dopt.backgroundColor));
 
-        final LinearLayoutManager lam = (LinearLayoutManager) _recyclerList.getLayoutManager();
-        final DividerItemDecoration dividerItemDecoration = new DividerItemDecoration(activity, lam.getOrientation());
-        _recyclerList.addItemDecoration(dividerItemDecoration);
+        // final LinearLayoutManager lam = (LinearLayoutManager) _recyclerList.getLayoutManager();
+        // final DividerItemDecoration dividerItemDecoration = new DividerItemDecoration(activity, lam.getOrientation());
+        // _recyclerList.addItemDecoration(dividerItemDecoration);
```

Comment:
Divider decoration not needed. The items do this themselves?

Reply:
Depends on how it's used. Generally the builtin Recyclerview decoration are better performant but in case there is a own one, of course bad to have multiple.